src/puppetlabs/ezbake/core.clj

```diff
@@ -880,6 +880,9 @@ Additional uberjar dependencies:
                            (deploy-snapshot lein-project)
                            (:version lein-project))
         reproducible? (not (System/getenv "EZBAKE_ALLOW_UNREPRODUCIBLE_BUILDS"))
+        ;; Do not copy ezbake config files and resources from the deps in this list.
```

Comment:
Are these just the files in the `resources` directory (or I guess whatever the specified `:resource-paths` are)? If so, maybe worth mentioning that?

Reply:
Yeah I think that's what it wound up being. I wasn't sure if there was more stuff at first.